src/background.js

```diff
@@ -230,6 +230,19 @@ background.updateBadge = function(props) {
   }
 };
 
+/**
+ * Checks for expired alarms
```

Comment:
Returns true if all alarms for the given `event` have expired, relative to the current time.